src/Angor/Client/Components/FounderProjectItem.razor

```diff
@@ -1,49 +1,144 @@
+                    </div>
+                </div>
+
+            </a>
+
+            <div class="card-body pb-0">
+
+
+
+                <div class="d-flex align-items-center mb-4">
+                    <span class="user-select-none">
+                        <Icon IconName="view" Height="24" Width="24"></Icon>
+                    </span>
+                    <div class="h-100 ms-3">
+                        <h5 class="mb-0 font-weight-bolder">
+                            <a href="/view/@FounderProject.ProjectInfo.ProjectIdentifier"> @FounderProject.Metadata.Name</a>
+                        </h5>
+                    </div>
+                </div>
+                <p class="mb-0 line-clamp-3">@(ConvertToMarkupString(StripHtmlTags(FounderProject.Metadata.About)))</p>
+
+
+
+            </div>
+            <div class="card-footer pt-0">
+                <hr class="horizontal light mt-3">
+
+                <a role="button" class="d-flex align-items-center btn btn-border w-100-m" href=@($"/view/{FounderProject.ProjectInfo.ProjectIdentifier}")>
+                    <span class="user-select-none">
+                        <Icon IconName="view-project" Height="24" Width="24"></Icon>
+                    </span>
+                    <div class="h-100 ms-3">
+                        <span class="mb-0 font-weight-bolder text-primary">
+                            View Project
+                        </span>
+                    </div>
+                </a>
+                @if (InvestmentRequests)
+                {
+                    <hr class="horizontal light mt-3">
+                    <a role="button" class="d-flex align-items-center btn btn-secondary w-100-m" href=@($"/signatures/{FounderProject.ProjectInfo.ProjectIdentifier}")>
+                        <span class="user-select-none">
+                            <Icon IconName="check-circle" Height="24" Width="24"></Icon>
+                        </span>
+                        <div class="h-100 ms-3">
+                            <span class="mb-0 font-weight-bolder">
+                                Approve signatures
+                            </span>
+                        </div>
+                    </a>
+                }
+
+            </div>
+        </div>
     </div>
-</div>
+ 
 
 @code {
 
     [Parameter]
     public FounderProject FounderProject { get; set; }
 
     public bool InvestmentRequests { get; set; }
-    
+
     protected override async Task OnInitializedAsync()
     {
         await RelayService.LookupSignaturesDirectMessagesForPubKeyAsync(FounderProject.ProjectInfo.NostrPubKey, FounderProject.LastRequestForSignaturesTime?.AddSeconds(1), 1,
             _ =>
             {
                 if (InvestmentRequests)
                     return;
-                
+
                 InvestmentRequests = true;
                 StateHasChanged();
             });
     }
+
+    public string StripHtmlTags(string input)
```

Comment:
Such methods should go in some helper class ( I guess they are not unique to this page)
So in the next PR please move them to their own class

Reply:
ok .